statement.md

```diff
@@ -0,0 +1,92 @@
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## 🚧 Draft Accessibility Statement for JupyterLab
+
+This is an accessibility statement from Jupyter accessibility contributors.
+
+### Measures to support accessibility
+
+Jupyter accessibility contributors take the following measures to ensure accessibility of JupyterLab:
+
+* Include accessibility as part of our mission statement.
+* Provide continual accessibility training for our community.
+* Assign clear accessibility goals and responsibilities.
+* Employ formal accessibility quality assurance methods.
+* Document changes, approaches, and improvements to the above methods and to JupyterLab itself.
+
+For more information on current efforts to improve JupyterLab's accessibility, visit the [JupyterLab accessibility grant roadmap](https://jupyter-a11y.netlify.app/roadmap/intro.html).
+
+### Conformance status
+
+The [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag) defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. JupyterLab is non conformant with WCAG 2.1 level AA. Non conformant means that the content does not meet the accessibility standard.
+
+![Picture accessibility compliance levels as mountains. The first peak to reach is AA. This includes both A and AA criteria. Behind it, in the mist, there is a larger peak, AAA. That one is not always completely reachable--Marie Guillaumet, Access42](https://stephaniewalter.design/wp-content/uploads/2022/05/stephaniewalter-sommets-de-l-accessibilite.jpg)
+
+*By [Stéphanie Walter](https://stephaniewalter.design) ([Source](https://stephaniewalter.design/blog/5-illustrations-to-understand-and-promote-accessibility/))*
+
+JupyterLab's accessibility does not exist in isolation. JupyterLab inherits much from the many projects it is built upon and its accessibility conformance may impact projects built off of JupyterLab or its components. The accessibility of this ecosystem is interlinked, so conformance may need to be resolved at different levels in order to impact JupyterLab positively.
+
+### Feedback and Formal complaints
+
+We welcome your feedback and formal complaints on the accessibility status of JupyterLab. Please let us know if you encounter accessibility barriers on JupyterLab:
+
+* [Write an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new)
+* [Write an issue on jupyterlab/jupyterlab](https://github.com/jupyterlab/jupyterlab/issues/new) and request it be labeled [tag:accessibility](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility)
+* If you are interested in being part of any potential research or organized feedback initiatives, please ___. Gauging community interest in user research participation helps us gain the support to make it happen.
```

Comment:
@trallard mentioned pointing people to the repo, but I think we need something a little more specific. I'm also unsure if a totally public place is the best since we may want to collect contact info. This is up for discussion.

I'm also happy to remove this section and more forward without it if it becomes too messy of a discussion.

```suggestion
* If you are interested in being part of any potential research or organized feedback initiatives, please [comment on this issue](). Gauging community interest in user research participation helps us gain the support to make it happen.
```
(Also oops to the accidental formatting I triggered. I'll fix that when we agree on what to do next.)

Reply:
I completely forgot - yes for now we can use your email and we should be able to request a Jupyter one too 🤔